nesta/packages/dq/tests/test_text.py

```diff
@@ -0,0 +1,34 @@
+import pandas as pd
+import numpy as np
+from collections import Counter
+from numpy.testing import assert_array_equal
+
+import pytest
+
+# from nesta.packages.dq.text import (string_length, string_counter)
+import sys
+sys.path.insert(0, '/mnt/c/Users/aotubusen/Google Drive/Documents/DS Projects/nesta/nesta/packages/dq')
+from text import (string_length, string_counter)
+
+@pytest.fixture

```

Comment:
If you add in the handling of missing values for `string_length` then it might be useful to have another fixture with some missing data too